src/Microsoft.ML.StandardLearners/Standard/OlsLinearRegression.cs

```diff
@@ -491,20 +491,6 @@ public static void Pptri(Layout layout, UpLo uplo, int n, Double[] ap)
                 }
             }
         }
-
-        [TlcModule.EntryPoint(Name= "Trainers.OrdinaryLeastSquaresRegressor", Desc = "Train an OLS regression model.", UserName = UserNameValue, ShortName = ShortName)]
-        public static CommonOutputs.RegressionOutput TrainRegression(IHostEnvironment env, Arguments input)
-        {
-            Contracts.CheckValue(env, nameof(env));
-            var host = env.Register("TrainOLS");
-            host.CheckValue(input, nameof(input));
-            EntryPointUtils.CheckInputArgs(host, input);
-
-            return LearnerEntryPointsUtils.Train<Arguments, CommonOutputs.RegressionOutput>(host, input,
-                () => new OlsLinearRegressionTrainer(host, input),
-                () => LearnerEntryPointsUtils.FindColumn(host, input.TrainingData.Schema, input.LabelColumn),
-                () => LearnerEntryPointsUtils.FindColumn(host, input.TrainingData.Schema, input.WeightColumn));
-        }
```

Comment:
Rather than just remove the entry-point, could we temporarily just remove the entire file from this repo since none of it is useful now, and only reintroduce it once we decide exactly what we're going to do about the underlying math library?

Reply:
Very well. @glebk is the one whose opinions on this matter are the ones that are most relevant. If he has reasoned through the implications of this (e.g., everything that uses entry-points will no longer have access to this) and has concluded there are no reasons for concerns, then I have no concerns either.